### ISSUE TITLE
Calling getContexts() does not return WEBVIEW contexts for iOS 13 (read device) using Appium 1.15.0

### ISSUE THREAD

**Issue by mikesalvia** [appium/appium#13668]
## The problem

When calling `getContexts` using an iOS 13 real device on 1.15.0 only `NATIVE_APP` is returned for hybrid apps, `WEBVIEW` contexts are returned in with 1.14.0

## Environment

* Appium version (or git revision) that exhibits the issue: 1.15.0
* Last Appium version that did not exhibit the issue (if applicable): 1.14.0
* Desktop OS/version used to run Appium: BrowserStack
* Node.js version (unless using Appium.app|exe): 10.16.3
* Npm or Yarn package manager: yarn
* Mobile platform/version under test: iOS
* Real device or emulator/simulator: Real device
* Appium CLI or Appium.app|exe: BrowserStack

## Details

When calling `getContexts` using an iOS 13 real device on 1.15.0 only `NATIVE_APP` is returned, this works with 1.14.0

## Link to Appium logs

Create a [GIST](https://gist.github.com) which is a paste of your _full_ Appium logs, and link them here.
Do _NOT_ paste your full Appium logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include Appium logs!

To be added

## Code To Reproduce Issue [ Good To Have ]

To be added

**Comment by mykola-mokhnach** [comment]
Have you tried appium@beta?

**Comment by mikesalvia** [comment]
Not yet, I will try there, it does work against simulators.

RESULTS
❗️ appium@1.15.0 *AND* iOS 13 Real Device (BrowserStack)
✅ appium@1.15.1 *AND* iOS 13 Simulator (Locally)

**Comment by mikesalvia** [comment]
@mykola-mokhnach Will investigate this more today and report findings.

**Comment by mikesalvia** [comment]
Trying `beta` now. Added some notes above.

**Comment by mikesalvia** [comment]
Starting to lean towards and issue with Xcode 11.2.2. I am going to hop on another machine that has Xcode 11.1 on it.

**Comment by mikesalvia** [comment]
@mykola-mokhnach I can only speculate but I am under an assumption that BrowserStack is using Xcode 11.2.2, will continue to gather data though.

**Comment by mikesalvia** [comment]
@mykola-mokhnach Retracted comments I had a code issue, but do still see the issue in Browserstaack will try with beta and real device locally.

**Comment by mikesalvia** [comment]
Looks like my device can only be updated to 12.4.3 so I can't do a local test on iOS 13.

**Comment by dannyfink** [comment]
@mikesalvia are you overriding the browserstack appium version capability in browserstack? It seem's that (at least for safari) that even if you set 1.15.0 its using 1.14.0 in favor of it.

**Comment by mikesalvia** [comment]
@dannyfink I am overriding with `1.15.0` using App Automate service, I see that with Automate `1.14.0` is always being spawned, but in App Automate it is another story.

**Comment by dannyfink** [comment]
Oh ok. I'm running into context switching issues myself for alert handling using 1.14.0 + iOS 13...I'm going to have to wait on them to have 1.1.5.0 available for "Automate" (which works on my local appium) before upgrading from iOS 12 to 13. 

I ran into this while doing some research. It's always awesome to see you on top of this stuff 👍

**Comment by mikesalvia** [comment]
:-) What kind of alert issue? I have noticed that `getSize()` behaves oddly in iOS 13 with Appium 1.14.0

**Comment by dannyfink** [comment]
`getAlertText()` fails...The /alert/text response comes back with an empty body. It works fine for iOS 12 using 1.14.0 as well as iOS13 using 1.15.1 (tested locally), but fails with iOS13 using 15.0

**Comment by mikesalvia** [comment]
You could use
`
function iOsAlertDialog(name="") {
  let locator = "-ios class chain:**/XCUIElementTypeAlert";
  locator = name !== "" ? locator + `[\`name = \"${name}\"\`]` : locator;
  return $(locator)
}
`
Instead of `getAlertText()` as a work around from `NATIVE_APP` context, of could you would need to develop one for Android as well.

**Comment by dannyfink** [comment]
@mikesalvia I was able to get it working with using a predicate on `XCUIElementTypeTextView`

**Comment by mykola-mokhnach** [comment]
To me it looks like a problem on BS side. Closed as not issue.

**Comment by mikesalvia** [comment]
@mykola-mokhnach I will see if I can get some more details on this if I don't see it resolved in BrowserStack when they update to 1.15.1.
